project-metrics/metrics_service/apis/travis.py

```diff
@@ -26,22 +28,36 @@ def __init__(self, status_code: int, err_msg: Text):
 class TravisApi(agithub_base.API):
   """Travis API interface."""
 
-  def __init__(self, *args, **kwargs):
+  # Default/maximum page size from the API.
+  TRAVIS_PAGE_SIZE = 25
```

Comment:
Place this here makes this an instance field, which is not necessarily what you want. There's some odd behaviours around this (most of them are irrelevant to this PR, but it's best to just place this at the module level)

Examples of unexpected behaviours:

```python
class A(object):
  F = 1

print(A.F)  # "1"

a1 = A()
a2 = A()

a1.F = 2
print(A.F, a1.F, a2.F)  # "1 2 1"

A.F = 3
print(A.F, a1.F, a2.F)  # "3 2 3"
```

Reply:
Done